sdcm/utils.py

```diff
@@ -196,6 +212,45 @@ def upload_file(cls, file_path):
             logger.debug("Unable to upload to S3: %s" % e)
             return ""
 
+    @classmethod
```

Comment:
why do you use @classmethod all over the class? did you mean `@staticmethod`?

Reply:
rewritten